Donky-Network/node-eventstore
# upgrade mongodb to be in line with version in core

**What's being changed**
update mongodb 

**Detailed explanation**
update mongodb to 3.3.8

**Why it's being changed**
to be in line with version used in core

**How to review / test this change**
add updated eventstore to core and update it in any service and check whether it works
